README.md

```diff
@@ -1,22 +1,20 @@
 # HIPIFY
+HIPIFY is a set of tools to translate CUDA source code into portable [HIP](https://github.com/ROCm-Developer-Tools/HIP) C++ automatically.
 
-### Tools to translate CUDA source code into portable HIP C++ automatically
-## Table of Contents
-
-<!-- toc -->
-
-- [hipify-clang](#clang)
-     * [Dependencies](#dependencies)
-     * [Usage](#hipify-clang-usage)
-     * [Building](#building)
-     * [Testing](#testing)
-     * [Linux](#linux)
-     * [Windows](#windows)
-- [hipify-perl](#perl)
-     * [Usage](#hipify-perl-usage)
-     * [Building](#hipify-perl-building)
-- [Supported CUDA APIs](#cuda-apis)
-- [Disclaimer](#disclaimer)
+## Documentation
+Information about HIPIFY and other user topics is found in the [HIPIFY documentation](https://rocmdocs.amd.com/projects/HIPIFY/en/latest/).
+The documentation source is browseable [here](docs/source/index.md).
+
+## Prerequisites
+The [ROCm website](https://docs.amd.com) describes how to set up the ROCm repositories and install the required platform dependencies.
+
+## Installing pre-built packages
+With the AMD ROCm package repositories installed, the `hipify` package can be retrieved from the system package manager. For example, on Ubuntu:
+
+```
+sudo apt-get update
+sudo apt-get install hipify
+```
```

Comment:
This subparagraph should be moved to the Linux paragraph.

Reply:
Fixed in latest [47e9926](https://github.com/ROCm-Developer-Tools/HIPIFY/pull/737/commits/47e9926058e7fe40d09fe4b083dff7b95296bcce)